$NetBSD: patch-Makefile,v 1.1 2017/09/04 09:44:41 leot Exp $

Instruct Makefile to properly handle also man pages and scripts via
corresponding INSTALL_{MAN,SCRIPT} in a similar vein of INSTALL_BIN.

--- Makefile.orig	2016-08-13 13:58:21.000000000 +0000
+++ Makefile
@@ -3,6 +3,8 @@ PERLSCRIPTS=vidir vipe ts combine zrun c
 MANS=sponge.1 vidir.1 vipe.1 isutf8.1 ts.1 combine.1 ifdata.1 ifne.1 pee.1 zrun.1 chronic.1 mispipe.1 lckdo.1 parallel.1 errno.1
 CFLAGS?=-O2 -g -Wall
 INSTALL_BIN?=install -s
+INSTALL_MAN?=install
+INSTALL_SCRIPT?=install
 PREFIX?=/usr
 
 ifneq (,$(findstring CYGWIN,$(shell uname)))
@@ -25,10 +27,10 @@ isutf8: is_utf8/*.c is_utf8/*.h
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL_BIN) $(BINS) $(DESTDIR)$(PREFIX)/bin
-	install $(PERLSCRIPTS) $(DESTDIR)$(PREFIX)/bin
+	$(INSTALL_SCRIPT) $(PERLSCRIPTS) $(DESTDIR)$(PREFIX)/bin
 
-	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	install $(MANS) $(DESTDIR)$(PREFIX)/share/man/man1
+	mkdir -p $(DESTDIR)$(PREFIX)/man/man1
+	$(INSTALL_MAN) $(MANS) $(DESTDIR)$(PREFIX)/man/man1
 
 check: isutf8
 	./is_utf8/test.sh
